lib/linguist/languages.yml

```
@@ -3306,6 +3306,30 @@ Lex:
   tm_scope: source.lex
   ace_mode: text
   language_id: 199
+LigoLANG-Cameligo:
```

From the [branding](https://ligolang.org/):
```suggestion
CameLIGO:
```